policy-controller/k8s/status/src/index.rs

```diff
@@ -398,3 +445,36 @@ fn accepted() -> k8s::Condition {
         type_: "Accepted".to_string(),
     }
 }
+
+fn resolved_refs() -> k8s::Condition {
+    k8s::Condition {
+        last_transition_time: k8s::Time(now()),
+        message: "".to_string(),
+        observed_generation: None,
+        reason: "ResolvedRefs".to_string(),
+        status: "True".to_string(),
+        type_: "ResolvedRefs".to_string(),
+    }
+}
+
+fn backend_not_found() -> k8s::Condition {
+    k8s::Condition {
+        last_transition_time: k8s::Time(now()),
+        message: "".to_string(),
+        observed_generation: None,
+        reason: "BackendNotFound".to_string(),
+        status: "False".to_string(),
+        type_: "ResolvedRefs".to_string(),
+    }
+}
+
+fn invalid_backend_kind() -> k8s::Condition {
+    k8s::Condition {
+        last_transition_time: k8s::Time(now()),
+        message: "".to_string(),
+        observed_generation: None,
+        reason: "InvalidKind".to_string(),
+        status: "False".to_string(),
+        type_: "ResolvedRefs".to_string(),
```

Comment:
nit, take it or leave it: I would probably make the repeated strings (e.g. "ResolvedRefs") constants, to guard against typos? but, there aren't any typos _yet_ and maybe we're not going to change them, so it's not a huge deal...

Reply:
I see the use for it. We might not change them soon but maybe it's better not to chance either... I also updated the Accepted ones so it's not awkward (where one set of conditions uses constants and the other doesn't).